qwandaq/src/main/java/life/genny/qwandaq/utils/EntityAttributeUtils.java

```diff
@@ -151,32 +150,32 @@ public Map<String, EntityAttribute> getAllEntityAttributesInParent(BaseEntity de
 	/**
 	 * Fetch an EntityAttribute value from the cache.
 	 *
-	 * @param pcm           The BaseEntity to fetch for
+	 * @param baseEntity    The BaseEntity to fetch for
 	 * @param attributeCode The Attribute code of the EntityAttribute to fetch
-	 * @return The corresponding EntityAttribute with embedded "Attribute", or null
-	 *         if not found.
+	 * @return The value of the corresponding entity attribute (decided by its DataType)
+	 * 
+	 * @throws ItemNotFoundException if the EntityAttribute could not be found
+	 * @see {@link EntityAttribute#getValue}
+	 * @see {@link EntityAttributeUtils#getEntityAttribute}
 	 */
-	public <T> T getValue(PCM pcm, String attributeCode) {
```

Comment:
Removed this as it is an unnecessary overload. (PCM extends BaseEntity, in the return, PCM is casted up back to BaseEntity. This same functionality is achieved by the overload on line 161)